src/kms/entry.rs

```diff
@@ -65,6 +65,11 @@ impl KeyEntry {
         self.name.as_str()
     }
 
+    /// Accessor for the key tags
+    pub fn tags_as_ref(&self) -> &Vec<EntryTag> {
```

Comment:
This could return a slice to avoid exposing implementation details: `self.tags.as_slice()`